src/main.ts

```diff
@@ -96,6 +96,8 @@ async function run() {
         await exec("git", ["clone", "--quiet", downstreamRepo, downstreamDir]);
 
         await exec("git", ["checkout", "-b", branchName], inDownstreamOptions);
+        await exec("git", ["submodule", "update", "--init"], inDownstreamOptions);
```

Comment:
I think that this will run in the downstream `pulumi-aws`. My understanding from the aws `Makefile` is that we want to run in `pulumi-aws/upstream`:

https://github.com/pulumi/pulumi-aws/blob/621af7ff4c7a233a2b2f335470760b99f19b774f/Makefile#L119-L123

Can you clarify?

Reply:
`git submodule update --init` can run anywhere since it reads the list of submodules from the Git index. In the aws Makefile, the command then also runs `git remote add` which needs to be inside the submodule folder.

Reply:
The latest test run shows the git checkout succeeding (with submodule), although I can't yet explain the `yarn` failure later.